tests/DoctrineMongoODMModuleTest/Doctrine/ConnectionFactoryTest.php

```diff
@@ -174,4 +175,25 @@ public function testConnectionStringWithOptionsShouldSetDefaultDB()
 
         self::assertEquals($dbName, $configuration->getDefaultDB());
     }
+
+    public function testEventManagerShouldSetEventManager()
```

Comment:
Trying to fit in with the existing naming which seems to be `test<Input><ConnectionBehaviour>`. That said I may have misunderstood the existing naming scheme, and I see the benefit of simplifying to avoid misunderstandings.